cpp/src/cylon/util/macros.hpp

```diff
@@ -57,7 +57,7 @@
     if ((arrow_table)->column(0)->num_chunks() > 1){            \
       const auto &res = (arrow_table)->CombineChunks((pool));   \
       RETURN_CYLON_STATUS_IF_ARROW_FAILED(res.status());        \
-      (arrow_table) = res.ValueOrDie();                         \
+      (arrow_table) = std::move(res).ValueOrDie();              \
```

Comment:
is there a reason for the parentheses around arrow_table variable?

Reply:
yeah, that a macros thing. It means to replace `(arrow_table)` site with the passed `arrow_table` param. (so you can pass something like `auto foo`)